mindocr/data/transforms/rec_abinet_transforms.py

```diff
@@ -0,0 +1,226 @@
+"""
+transform for text recognition tasks.
+"""
+import copy
+import logging
+import random
+import re
+import warnings
+
+import cv2
+import numpy as np
+import PIL
+import six
```

Comment:
Do not use `six`! `six` is needed for backward compatibility with python 2.x. Our minimum supported version of python is 3.7!